src/am_map.c

```diff
@@ -2358,61 +2358,60 @@ void AM_ColorPreset(void)
+             "Show automap pointer coordinates in non-follow mode");
   M_BindBool("map_secret_after", &map_secret_after, NULL, false, ss_auto, wad_no,
-             "1 to not show secret sectors till after entered");
+             "Don't highlight secret sectors on the automap until after entering them");
   M_BindNum("map_keyed_door", &map_keyed_door, NULL,
             MAP_KEYED_DOOR_COLOR, MAP_KEYED_DOOR_OFF, MAP_KEYED_DOOR_FLASH,
             ss_auto, wad_no,
-            "Keyed doors are colored (1) or flashing (2) on the automap");
+            "Color key-locked doors on the automap (1 = Static; 2 = Flashing)");
   M_BindBool("map_smooth_lines", &map_smooth_lines, NULL, true, ss_auto,
-             wad_no, "1 to enable smooth automap lines");
+             wad_no, "Smooth automap lines");
 
   M_BindNum("mapcolor_preset", &mapcolor_preset, NULL, 1, 0, 2, ss_auto, wad_no,
-            "Automap color preset (0 = Vanilla Doom, 1 = Boom (default), "
-            "2 = ZDoom)");
+            "Automap color preset (0 = Vanilla Doom; 1 = Boom; 2 = ZDoom)");
 
 #define BIND_CR(name, v, help) \
   M_BindNum(#name, &name, NULL, (v), 0, 255, ss_none, wad_yes, help)
 
-  BIND_CR(mapcolor_back, 247, "Color used as background for automap");
-  BIND_CR(mapcolor_grid, 104, "Color used for automap grid lines");
-  BIND_CR(mapcolor_wall, 23, "Color used for one side walls on automap");
-  BIND_CR(mapcolor_fchg, 55, "Color used for lines floor height changes across");
-  BIND_CR(mapcolor_cchg, 215, "Color used for lines ceiling height changes across");
+  BIND_CR(mapcolor_back, 247, "Color used for the automap background");
+  BIND_CR(mapcolor_grid, 104, "Color used for grid lines");
+  BIND_CR(mapcolor_wall, 23, "Color used for one-sided walls");
+  BIND_CR(mapcolor_fchg, 55, "Color used for lines across which floor height changes");
+  BIND_CR(mapcolor_cchg, 215, "Color used for lines across which ceiling height changes");
   BIND_CR(mapcolor_clsd, 208, "Color used for lines denoting closed doors, objects");
-  BIND_CR(mapcolor_rkey, 175, "Color used for red key sprites");
-  BIND_CR(mapcolor_bkey, 204, "Color used for blue key sprites");
-  BIND_CR(mapcolor_ykey, 231, "Color used for yellow key sprites");
+  BIND_CR(mapcolor_rkey, 175, "Color used for red-key sprites");
+  BIND_CR(mapcolor_bkey, 204, "Color used for blue-key sprites");
+  BIND_CR(mapcolor_ykey, 231, "Color used for yellow-key sprites");
   BIND_CR(mapcolor_rdor, 175, "Color used for closed red doors");
   BIND_CR(mapcolor_bdor, 204, "Color used for closed blue doors");
   BIND_CR(mapcolor_ydor, 231, "Color used for closed yellow doors");
   BIND_CR(mapcolor_tele, 119, "Color used for teleporter lines");
   BIND_CR(mapcolor_secr, 252, "Color used for lines around secret sectors");
-  BIND_CR(mapcolor_revsecr, 112, "Color used for lines around revealed secret sectors");
+  BIND_CR(mapcolor_revsecr, 112, "Color used for lines around entered secret sectors");
```

Comment:
What's wrong with "revealed"?

Reply:
I think in the Doom context, a secret is considered as revealed if the secret sector had been entered. That's also the moment when the centered message appears on screen. The moment when it appears on the automap is irrelevant.